src/org/rascalmpl/library/lang/html5/DOM.rsc

```diff
@@ -428,13 +425,17 @@ str nodeToString(str n, set[HTML5Attr] attrs, list[value] kids) {
       return s;
 }
   
-public HTML5Node example 
+private HTML5Node example 
```

Comment:
Is this some kind of inline documentation?

Reply:
I don't know yet. Have to ask Tijs. I think it's test code